libs/layout/src/lib/const/drag-and-drop/drag-and-drop-message.const.ts

```diff
@@ -0,0 +1,106 @@
+	},
+	es: {
+		selected: '{{#item}} elegido.',
+		deselected: '{{#item}} rechazado.',
+		reordered: '{{#item}} fue reorganizado de la posición {{#from}} a {{#to}}.',
+		moved: '{{#item}} fue trasladado de {{#from}} a {{#to}}.',
+		cancelled: 'El desplazamiento de {{#item}} estuvo anulado.',
+		item: 'objeto',
+		container: 'lista',
+		description:
+			'Para trasladar los objetos en esta sección, diríjase al objeto con la tecla Tab. Al tocar el botón Enter o el botón de Espacio puede seleccionar el objeto. Con las flechas puede trasladar el objeto.',
+	},
+	pt: {
+		selected: '{{#item}} selecionado.',
+		deselected: '{{#item}} desselecionado.',
+		reordered: '{{#item}} foi movido da posição {{#from}} para {{#to}}.',
+		moved: '{{#item}} foi movido de {{#from}} para {{#to}}.',
+		cancelled: 'A movimentação de {{#item}} foi cancelada.',
+		item: 'item',
+		container: 'lista',
+		description:
+			'Para mover itens nesta seção, navegue até um item usando a tecla Tab. Pressione Enter ou Barra de Espaço para selecionar um item. Use as teclas de seta para mover o item.',
+	},
+	tr: {
+		selected: '{{#item}} seçildi.',
+		deselected: '{{#item}} seçimi kaldırıldı.',
+		reordered: '{{#item}} {{#from}} konumundan {{#to}} konumuna taşındı.',
+		moved: '{{#item}} {{#from}} konumundan {{#to}} konumuna taşındı.',
+		cancelled: '{{#item}} taşınması iptal edildi.',
+		item: 'öğe',
+		container: 'liste',
+		description:
+			'Bu bölümdeki öğeleri taşımak için Tab tuşunu kullanarak bir öğeye gidin. Bir öğeyi seçmek için Enter veya Boşluk tuşuna basın. Öğeyi taşımak için ok tuşlarını kullanın.',
+	},
+	de: {
+		selected: '{{#item}} ausgewählt.',
+		deselected: '{{#item}} abgewählt.',
+		reordered: '{{#item}} wurde von Position {{#from}} nach Position {{#to}} verschoben.',
+		moved: '{{#item}} wurde von {{#from}} nach {{#to}} verschoben.',
+		cancelled: 'Das Verschieben von {{#item}} wurde abgebrochen.',
+		item: 'Eintrag',
+		container: 'Liste',
+
+		description:
+			'Um Einträge in diesem Teil zu verschieben, navigiere mit der Tab-Taste zu einem Eintrag. Mit Drücken der Eingabe- oder Leerzeichentaste wählst du einen Eintrag aus. Mit den Pfeiltasten verschiebst du den Eintrag.',
+	},
+	ku: {
+		selected: '{{#item}} hate hilbijartin.',
+		deselected: '{{#item}} hate jêbirin.',
+		reordered: '{{#item}} ji cîhê {{#from}} ve ji cîhê {{#to}} ve veguhast.',
+		moved: '{{#item}} hate veguhastin ji {{#from}} ve {{#to}}.',
+		cancelled: 'Veguhastina {{#item}} hate betal kirin.',
+		item: 'şanî',
+		container: 'lista',
+		description:
+			'Ji bo veguhastina şanîyan di vê beşê de, bi bikaranîna tuşeya Tab ve her şanîyekê biguherîne. Ji bo hilbijartina şanîyekê, bişkojka Enter an Space bikirtînin. Bi bikaranîna tuşeya şilî şanîyê biguherînin.',
+	},
+	ru: {
```

Comment:
Just a reminder not to merge this until @SergueySHD validates these translations.